reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt

```diff
@@ -94,6 +94,7 @@ class ReportTableModelMapper(private val resolutionProvider: ResolutionProvider)
             val allIds = sortedSetOf(project.id)
             allIds += project.collectDependencies()
 
+            val projectIssues = project.collectIssues()
```

Comment:
Commit message:
- "in order to [collect] the issues in O(n) i[ns]tead [of] O(n²) time"
- "due to a[n] unusual large amount [of] scopes"

"was observed to take about one hour" -> so, how fast is it now?

Reply:
Done.

Reply:
Also commit message:
* "Use Project.collectIssue**s**()"
* Still: "amount **of** scopes"

Reply:
Done.